src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -719,29 +707,19 @@
     }
   }
 
-  .datagrid-footer-select.checkbox {
-    display: block;
-    line-height: inherit;
-    flex: 0 0 (1.5rem - (2 * $clr-rem-1px)); // can't use calc() in flex. #thanksIE!
-    margin-right: 0.083333rem; // Space for the instance when a hideable column toggle is next to it.
-    align-self: flex-start;
-    label {
-      color: unset;
-      cursor: default;
-      opacity: 1;
-      line-height: inherit;
-    }
-
-    input[type='checkbox'] + label {
-      &::before,
-      &::after {
-        top: 0.333333rem;
-      }
+  .clr-form-control-disabled .datagrid-footer-select.clr-checkbox-wrapper input[type='checkbox']:checked + label {
+    top: calc(1rem / 6);
```

Comment:
Did this get tested in IE11? (referencing line #725 from the deleted code)

Reply:
`calc()` can be used in IE11 for `top` so should be ok. will do some testing to confirm

Reply:
Sounds good.